language/move-bytecode-verifier/src/absint.rs

```diff
@@ -54,7 +56,8 @@ pub trait TransferFunctions {
         instr: &Bytecode,
         index: CodeOffset,
         last_index: CodeOffset,
-    ) -> Result<(), Self::Error>;
+        meter: &mut impl Meter,
+    ) -> PartialVMResult<()>;
```

Comment:
I see now why you did this. I do not remember the details, but if we would not do that, it would be harder to bubble up the error from `meter` which is a VM result. Perhaps you can switch it back?